test/integration/provider/rti.go

```diff
@@ -0,0 +1,149 @@
+package provider
+
+import (
+	"fmt"
+
+	provider "github.com/gardener/machine-controller-manager-provider-azure/pkg/azure"
+	v1alpha1 "github.com/gardener/machine-controller-manager/pkg/apis/machine/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	// cluster tag and cluster values are specifically used for integration test
+	// in case the test is run against non seed cluster then the supplied MachineClass
+	// is expected to have these tags set so that the machines from this suite won't be
+	// orphan collected.
+	clusterTag      = "mcm-integration-test"
+	clusterTagValue = "true"
```

Comment:
Update the tag key and value

```suggestion
	clusterTag      = "kubernetes.io-role-integration-test"
	clusterTagValue = "1"
```